### DISCUSSION TITLE
Feature Request : Partially Funded Mutual Territories

### DISCUSSION THREAD

**Discussion by ghost** [stackernews/stacker.news#1876] [Ideas]
Issue : 1065
     Title : Partially Funded Mutual Territories
     Author : Radentor
     Status : Proposed
     Type : Feature
     Created : 2024-04-14

# Abstract
This issue proposes a change in how territories are funded from single-funded only to be possibly funded by multiple stackers.

# Motivation
Over time, most of the territories were left abandoned by it's founder because they simply could not maintain it any longer by him/her-self.  It's a no brainer to say that territory it's expected to last longer if funded by multiple stackers .

I propose 2 new model of funding SN territories, first model is similar to IPO in concept and second model is literally just a donation. The current crowdfunding for territory is arguably inefficient due to 10% zap fee and also involves trust for founder not to misuse the donated sats.

In the first model, before starting a territory, founder can add up to 4 cofounders with control of how much is their share of fee. A stacker can reject invitation to become a territory cofounder at their will, but if they agree, they will be subject to territory fee and territory revenue. Another good thing is that 3M sats one-time fee might sound less scary.

In the second model, ownership of territory remains intact to it's founder. The current timer for territory will be shown along with a progress bar indicating it's remaining cost to pay for next month (if yet to be paid). Then on the territory "info tab" there should be a a `Donate` button alongside the previous progress bar for stacker to donate.

# Pros & Cons
- Pros :
1. Cheaper cost to fund & sustain a territory
2. Trustless crowdfunding (second model)
3. More efficient & easier crowdfunding (second model)

- Cons :
1. First model can be subject to "unregistered security" issue (but revenue-sharing can be disabled)

# Earlier Mentions
- https://stacker.news/items/339744?commentId=340062
- https://stacker.news/items/353027?commentId=353043
- https://stacker.news/items/353027?commentId=353945
- https://stacker.news/items/359805?commentId=359811
- https://stacker.news/items/412066?commentId=412301

**Comment by h0dlr** [comment]
https://stacker.news/items/614066
